apps/fluent-tester/src/TestComponents/TextExperimental/StandardUsage.tsx

```diff
@@ -4,12 +4,17 @@ import { Text } from '@fluentui-react-native/experimental-text';
 import { Stack } from '@fluentui-react-native/stack';
 import { stackStyle } from '../Common/styles';
 import { HOMEPAGE_EXPERIMENTAL_TEXT_BUTTON } from './consts';
+import { testProps } from '../Common/TestProps';
 
 export const StandardUsage: React.FunctionComponent = () => {
   return (
     <View>
       <Stack style={stackStyle} gap={5}>
-        <Text variant="captionStandard" testID={HOMEPAGE_EXPERIMENTAL_TEXT_BUTTON}>
```

Comment:
Same here except [TextExperiementalTest.tsx](https://github.com/microsoft/fluentui-react-native/blob/main/apps/fluent-tester/src/TestComponents/TextExperimental/TextExperimentalTest.tsx#L14) already has a testID set so I think we can just remove the testID here